### PR TITLE
improve exeption message

### PR DESCRIPTION
**Pull request by ronfroy**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

**Comment by ronfroy** [comment]
@teohhanhui done in another PR

File: src/Bridge/Symfony/Routing/IriConverter.php [hunk 1]
```diff
@@ -159,12 +159,22 @@ public function getSubresourceIriFromResourceClass(string $resourceClass, array
...
+     * @throws InvalidArgumentException
      *
      * @return string[]
      */
-    private function generateIdentifiersUrl(array $identifiers): array
+    private function generateIdentifiersUrl(array $identifiers, string $resourceClass): array
     {
+        if (0 === \count($identifiers)) {
+            throw new InvalidArgumentException(sprintf(
+                'You forgot to defined an identifier property for the resource of type "%s"',
```
Comment (jonag) [inline src/Bridge/Symfony/Routing/IriConverter.php]: define~d~

Comment (ronfroy) [reply src/Bridge/Symfony/Routing/IriConverter.php]: @jonag done

File: src/Bridge/Symfony/Routing/IriConverter.php [hunk 2]
```diff
@@ -159,12 +159,22 @@ public function getSubresourceIriFromResourceClass(string $resourceClass, array
...
+     * @throws InvalidArgumentException
      *
      * @return string[]
      */
-    private function generateIdentifiersUrl(array $identifiers): array
+    private function generateIdentifiersUrl(array $identifiers, string $resourceClass): array
     {
+        if (0 === \count($identifiers)) {
+            throw new InvalidArgumentException(sprintf(
+                'You forgot to define an identifier property for the resource of type "%s"',
```
Comment (teohhanhui) [inline src/Bridge/Symfony/Routing/IriConverter.php]: > No identifiers defined for resource of type "%s"

Comment (ronfroy) [reply src/Bridge/Symfony/Routing/IriConverter.php]: @teohhanhui done

**Comment by teohhanhui** [comment]
@ronfroy I'm thinking it'd be good to have a test for this to prevent regressions in the future. Do you think you could add it before we merge? :smile:

**Comment by ronfroy** [comment]
@teohhanhui done

**Comment by ronfroy** [comment]
@teohhanhui merge?

**Comment by dunglas** [comment]
Thanks @ronfroy

**Pull request merged**